_config.yml

```
@@ -1 +1,3 @@
-theme: jekyll-theme-hacker
\ No newline at end of file
+theme: jekyll-theme-midnigh
```

change ```suggestion
midnigh
``
to 
```suggestion
midnight
```